src/polyfills.browser.ts

```diff
@@ -26,7 +26,7 @@ import 'core-js/es6/typed';
 import 'core-js/es6/reflect';
 // see issue https://github.com/AngularClass/angular2-webpack-starter/issues/709
 // import 'core-js/es6/promise';
-
+import 'pact-web';
```

Comment:
Sigh :) So in this world of amazing web development you have more than 1 place where you define your dependencies? (also `spec-bundle.js`).